src/toolkits/object_detection/object_detector.cpp

```diff
@@ -413,6 +413,13 @@ void object_detector::train(gl_sframe data,
                             variant_type validation_data,
                             std::map<std::string, flexible_type> opts)
 {
+  auto use_update_metrics_iter = opts.find("use_update_metrics");
+  if (use_update_metrics_iter == opts.end()) {
+    log_and_throw("Expected option \"use_update_metrics\" not found.");
+  }
```

Comment:
Let's not require the option. We can have a usable default value here, unlike `mlmodel_path`